utils/config/daos_agent.yml

```diff
@@ -60,6 +60,12 @@
 ## default: INFO
 #control_log_mask: DEBUG
 
+## Disable the agent's internal cache. If set to true, the agent will query the
+## access point directly every time a client requests server rank URIs.
+#
+## default: false
+#disable_cache: true
```

Comment:
Minor nit, not sure if it's worth reworking, though. The name of this parameter doesn't specify which cache is being disabled, and to a naive reader might imply that there is only one cache to be disabled. In fact, there are now two separate caches, right?

I think we could either add parameters to allow individual control of cache behaviors, or maybe just use this as a single "big hammer" to disable all agent caching. I'm kind of inclined to take the latter approach, and leave fine-grained control to the environment variables. In that case, we'd probably want to rename this to `disable_caching` or `disable_caches` and update the comment.

Reply:
Yeah, let's go with the big hammer. Thanks.